lib/gooddata/helpers/splunk_helper.rb

```diff
@@ -0,0 +1,51 @@
+require 'rest-client'
+require 'nokogiri'
+
+
+require_relative 'global_helpers'
+
+module GoodData
+  module Helpers
+    module SplunkHelper
+      class << self
+        @@host = 'https://localhost:8089'
+        @@paths = {
+          :login => '/services/auth/login?cookie=1',
+          :insert => '/services/receivers/simple'
+        }
+
+        def post(url_token, data, session_key = '', query = '')
+          return RestClient::Request.execute( :method => 'post', :url => @@host+@@paths[url_token]+"?"+query, :payload => data, :verify_ssl => false, :headers => {
+            "Authorization" => "Splunk "+session_key
+          })
+        end
+
+        def get(url_token, session_key = '', query = '')
+          return RestClient::Request.execute( :method => 'get', :url => @@host+@@paths[url_token]+"?"+query, :verify_ssl => false, :headers => {
+            "Authorization" => "Splunk "+session_key
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.